README.md

```diff
@@ -1,5 +1,6 @@
 # The Shopcarts Service
 
+[![Build Status](https://github.com/CSCI-GA-2820-SU23-001/shopcarts/actions/workflows/ci.yml/badge.svg)](https://github.com/CSCI-GA-2820-SU23-001/shopcarts/actions)
```

Comment:
Hi Wan-Yu thanks for review! I go check prof's demo on Codecov, it seems like we need to activate 'actions' on GitHub first so that the repo can be seen in Codecov and I can get the token then. So i am thinking to merge the CI badge first.

Reply:
ohh I see! thank you for double check on this 👍🏻